### PR TITLE
Documentation/cache-only-leaders

### PR DESCRIPTION
**Pull request by Simran-B**
Docs PR: https://github.com/arangodb/docs/pull/1340

Backport 3.10: https://github.com/arangodb/arangodb/pull/18715
Partial backport 3.9: https://github.com/arangodb/arangodb/pull/18716

### CODE REVIEW

**Comment by KVS85** [comment]
@Simran-B backport to 3.10 is required.

**Comment by Simran-B** [comment]
@KVS85 Yes, TBD = to be done (after the devel changes are reviewed)

File: Documentation/DocuBlocks/Rest/Indexes/post_api_index_inverted.md [hunk 1]
```diff
@@ -87,14 +87,22 @@ Default: the value defined by the top-level `trackListPositions` option.
 @RESTSTRUCT{cache,post_api_index_inverted_fields,boolean,optional,}
 Enable this option to always cache the field normalization values in memory
 for this specific field. This can improve the performance of scoring and
-ranking queries.
+ranking queries. Otherwise, these values are memory-mapped and it is up to the
+operating system to load them from disk into memory and to evict them from memory.
```
Comment (Dronplane) [inline Documentation/DocuBlocks/Rest/Indexes/post_api_index_inverted.md:91]: Also here we could mention fields processed with Geo analyzer. There is also auxilarry data to run geo SEARCH and also could be cached.

**Review by Dronplane — APPROVED** [review]
LGTM. One comment.

**Review by KVS85 — APPROVED** [review]

**Pull request merged**